src/game-support/planet-coaster.md

```diff
@@ -0,0 +1,15 @@
+# Planet Coaster
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Gold installs=Yes opens=Yes}}
```

Comment:
If this doesn't run in current version it should be Garbage not Gold

Reply:
Kind of, i thought that using a specific version could be seen as configuration, so with correct version it runs perfectly fine. Should i change this to Garbage anyway?

Reply:
I would say change to garbage yeah, newly added games should always be compatible with the current version of Whisky. If it works currently then you can change it to Gold.